<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>TGo4LogSettings</class>
  <widget class="QDialog" name="TGo4LogSettings" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>284</width>
        <height>192</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Set GUI Logging behaviour</string>
    </property>
    <property name="sizeGripEnabled" >
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
      <item row="4" column="0" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
              <property name="autoDefault" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="3" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QRadioButton" name="LogfileWrite" >
              <property name="text" >
                <string>writing to file:</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Enable logging to file.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="LogfileName" />
          </item>
          <item>
            <widget class="QToolButton" name="FileNameLog" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize" >
                <size>
                  <width>30</width>
                  <height>25</height>
                </size>
              </property>
              <property name="maximumSize" >
                <size>
                  <width>30</width>
                  <height>25</height>
                </size>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="icon" >
                <iconset>:/icons/findfile.png</iconset>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Browse for output file.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="2" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QRadioButton" name="LogfilePrint" >
              <property name="text" >
                <string>printing on GUI shell window</string>
              </property>
              <property name="toolTip" stdset="0" >
                <string>Enable logging in GUI shell window.</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Horizontal Spacing2_2" >
              <property name="sizeHint" >
                <size>
                  <width>110</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item row="1" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="IgnoreLabel" >
              <property name="text" >
                <string>Priority:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="IgnoreCombo" >
              <property name="toolTip" stdset="0" >
                <string>Priorities below selected one are included.</string>
              </property>
              <item>
                <property name="text" >
                  <string>Debugs</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Infos</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Warnings</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Errors</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="QLabel" name="ModeLabel" >
              <property name="text" >
                <string>Logging mode:</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QComboBox" name="ModeCombo" >
              <property name="toolTip" stdset="0" >
                <string>Defines when logging shall be done.</string>
              </property>
              <item>
                <property name="text" >
                  <string>On demand</string>
                </property>
              </item>
              <item>
                <property name="text" >
                  <string>Automatic</string>
                </property>
              </item>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>TGo4LogSettings</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>TGo4LogSettings</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>FileNameLog</sender>
      <signal>clicked()</signal>
      <receiver>TGo4LogSettings</receiver>
      <slot>LogfileDialog()</slot>
    </connection>
    <connection>
      <sender>LogfileWrite</sender>
      <signal>toggled(bool)</signal>
      <receiver>LogfileName</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>LogfileWrite</sender>
      <signal>toggled(bool)</signal>
      <receiver>FileNameLog</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
